src/draw/shapes.typ

```diff
@@ -556,8 +556,20 @@
 /// line((-1.5, 0), (1.5, 0))
 /// line((0, -1.5), (0, 1.5))
 /// line((-1, -1), (-0.5, 0.5), (0.5, 0.5), (1, -1), close: true)
-/// ```) 
-/// 
+/// ```)
+///
+/// If the first or last coordinate given is the name of an element,
+/// the intersection of that element and a line from the first or last
+/// two coordinates given is used as coordinate. This is useful to
```

Comment:
What about "If the first or last points are given as the name of an element, that has a "default" anchor, the intersection of that element's border and the line."